core/src/test/resources/sql/misc.iq

```diff
@@ -378,14 +378,14 @@ where exists (select 1 from "hr"."emps" where "empid" < 0);
 (0 rows)
 
 !ok
-EnumerableCalc(expr#0..1=[{inputs}], deptno=[$t0])
+EnumerableCalc(expr#0..1=[{inputs}], deptno=[$t1])
   EnumerableNestedLoopJoin(condition=[true], joinType=[inner])
```

Comment:
Some join factors are swapped because when rule applying sequence changes, the id of join  factors also changed. And EnumerableNestedLoopJoin has a special handling that it prefers left branch having a lower id than the right branch.